forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -1594,6 +1594,24 @@ public static void addTriggerAbility(final KeywordInterface inst, final Card car
             trigger.setOverridingAbility(AbilityFactory.getAbility(transformEff, card));
 
             inst.addTrigger(trigger);
+        } else if (keyword.startsWith("Offspring")) {
+            final String[] k = keyword.split(":");
+            final Cost cost = new Cost(k[1], false);
+            String costDesc = cost.toSimpleString();
+            if (!cost.isOnlyManaCost()) {
+                costDesc = "—" + costDesc;
+            }
+
+            final String trigStr = "Mode$ ChangesZone | Destination$ Battlefield " +
+                    "| ValidCard$ Card.Self+linkedCastSA | Secondary$ True " +
+                    "| TriggerDescription$ Offspring " + costDesc + " (" + inst.getReminderText() + ")";
+
+            final String effect = "DB$ CopyPermanent | Defined$ TriggeredCard | NumCopies$ 1 | SetPower$ 1 | SetToughness$ 1";
```

Comment:
should be LKICopy?

Reply:
i copied from Squad ... maybe that needs LKICopy too